test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java

```diff
@@ -82,22 +82,6 @@ public class CheckCipherSuites {
         "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256",
         "TLS_DHE_DSS_WITH_AES_128_CBC_SHA256",
 
-        // AES_256(GCM) - not forward screcy
```

Comment:
Can you please fix the rest of the"screcy" typos in this file?

Reply:
Fixed.

Reply:
Thanks.